File: dotnet/Templates/Microsoft.tvOS.Templates/tvos/csharp/ViewController.cs
using ObjCRuntime;

namespace tvOSApp1;

public partial class ViewController : UIViewController {
	protected ViewController (NativeHandle handle) : base (handle)
	{
		// This constructor is required if the view controller is loaded from a xib or a storyboard.
		// Do not put any initialization here, use ViewDidLoad instead.
	}
}

File: dotnet/Templates/Microsoft.tvOS.Templates/tvos/visualbasic/ViewController.vb
Imports ObjCRuntime
Imports UIKit

Namespace tvOSApp1
	Public Partial Class ViewController
		Inherits UIViewController
		Protected Sub New(ByVal handle As NativeHandle)
			MyBase.New(handle)
			' This constructor is required if the view controller is loaded from a xib or a storyboard.
			' Do not put any initialization here, use ViewDidLoad instead.
		End Sub
	End Class
End Namespace
